report/test/renderer/report-utils-test.js

```diff
@@ -98,6 +98,9 @@ describe('util helpers', () => {
       it('corrects underscored `notApplicable` scoreDisplayMode', () => {
         const clonedSampleResult = JSON.parse(JSON.stringify(sampleResult));
 
+        // Remove entity classification to be able to compare to sample_v2 results.
+        delete clonedSampleResult.entities;
```

Comment:
To check my understanding, this is to prevent us injecting a bunch of `entity` values into our audits which would mess with the diff at the end of this test.

If that's true, can you elaborate on that in the comment.

Reply:
Thats correct. I'll update the comment.